src/events/components/SearchModule/SelectMultiple.tsx

```diff
@@ -1,25 +1,59 @@
 import React, { ChangeEvent, FC } from 'react';
-import { EventTypeEnum, getEventType } from '../../models/Event';
+import Select from 'react-select';
+import makeAnimated from 'react-select/lib/animated';
+import { EventTypeEnum, getEventColor, getEventType } from '../../models/Event';
+import style from './search.less';
 
 const selectItems = () => {
   const eventTypeNumberList = Array(Object.keys(EventTypeEnum).length / 2 - 1)
     .fill(1)
     .map((x, y) => x + y);
+  return eventTypeNumberList.map((eventType) => ({ value: eventType, label: getEventType(eventType) }));
+};
 
-  return eventTypeNumberList.map((eventType) => (
-    <option key={eventType} value={eventType}>
-      {getEventType(eventType)}
-    </option>
-  ));
+const optionStyles = {
+  control: (styles: any) => ({ ...styles, backgroundColor: 'white' }),
```

Comment:
`styles` (and other input variables) probably have a type. They should _not_ be `any`

Reply:
Yup, it's `CSSProperties`